### DISCUSSION TITLE
Complete contributing page

### DISCUSSION THREAD

**Discussion by yangrong688** [apache/kyuubi#439] [General]
Kyuubi's contributing page are just some guidelines and not detailed implementation steps, we should complete contributing page to help people better contribute to community. Refer to [spark contributing page](http://spark.apache.org/contributing.html), we should add more details in contributing page like pr process / code review process / code style guide etc.

**Comment by yaooqinn** [comment]
LGTM, can you help to make our own page?

**Reply by yangrong688** [reply]
I'd like to make some contribute if I have time.

**Reply by yaooqinn** [reply]
thanks for  driving this
